### PR TITLE
Embed layer test

### PR DESCRIPTION
**Pull request by nworr**
Add new project with embed layer
and playwright test

Ticket : #3326  

Funded by 3liz

### CODE REVIEW

File: tests/end2end/cypress/integration/projects_embed_layer-ghaction.js [hunk 1]
```diff
@@ -0,0 +1,14 @@
+// projects_filename.js created with Cypress
```
Comment (nboisteault) [inline tests/end2end/cypress/integration/projects_embed_layer-ghaction.js:1]: Please just put a test for playwright.

File: tests/end2end/playwright/startup.spec.js [hunk 2]
```diff
@@ -40,4 +40,13 @@ test.describe('Startup', () => {
      
     await expect( page.locator('#layer-quartiers')).toHaveClass(/liz-layer/);
   });
+
+  test('Projects with embed layer can load', async ({page}) => {
+    const url_dots = '/index.php/view/map/?repository=testsrepository&project=embed_layer';
```
Comment (nboisteault) [inline tests/end2end/playwright/startup.spec.js:45]: Why `url_dots` variable name?

File: tests/end2end/playwright/startup.spec.js [hunk 3]
```diff
@@ -40,4 +40,13 @@ test.describe('Startup', () => {
...
+    await page.goto(url_dots, { waitUntil: 'networkidle' });
+    await page.$eval("#map, #map *", el => el.style.visibility = 'visible');
```
Comment (nboisteault) [inline tests/end2end/playwright/startup.spec.js:47]: Why hiding other elements?

**Review by nboisteault — CHANGES_REQUESTED** [review]

**Comment by nboisteault** [comment]
@nworr should it be reopened?

**Comment by nworr** [comment]
IMHO, the project file is way too simple to ensure embed layers actually works,  there may problematic use case that need to be checked ( user-filtered layers, mutiple projections, ...).

**Pull request closed without merging**